### DISCUSSION TITLE
Don't switch to preview pane after rendering

### DISCUSSION THREAD

**Discussion by dense-set** [quarto-dev/quarto-cli#9226] [Q&A] [answered]
### Description

Whenever I render a `.qmd` file in VSCode, the cursor goes to the preview pane. This is annoying for someone like me who does a lot of back and forth in editing and previewing. After each render, I need to bring the cursor to the original file manually. Is there a way for the cursor not to go to the preview pane after each render?

**Comment by mcanouil** [comment, marked as answer]
- https://github.com/quarto-dev/quarto/issues/361

PS: VSCode extension repository is <https://github.com/quarto-dev/quarto/>.
